.circleci/main.yml

```diff
@@ -1447,19 +1447,36 @@ jobs:
     steps:
       - checkout:
           path: ~/project
+      - install-minikube
+      - install-helm
+      - install-kubectl
+      - start-minikube
       - install-poetry
       - run: #Note: --no-ansi https://github.com/python-poetry/poetry/issues/7184
           name: Install Python Dependencies with Poetry
           command: poetry --no-ansi install
-      - install-python-test-deps
-
-      # export PACH_PYTHON_ENTERPRISE_CODE=${ENT_ACT_CODE}
-      # export export PACH_PYTHON_AUTH_TOKEN=iamroot
+      - wait-for-minikube
+      - run:
+          name: Install pach
+          command: |
+            helm install pachd ../etc/helm/pachyderm --set deployTarget=LOCAL,pachd.image.tag=${CIRCLE_SHA1},proxy.service.type=NodePort
```

Comment:
will this job also run at release time? because it will fail on console since it will not exist yet and there will be no `CIRCLE_SHA1` only a `CIRCLE_TAG`

Reply:
I'll make a note to address this in #8906